File: FirstAndLastDigit.java

import java.util.Scanner;

class FirstAndLastDigit
{
    public static void main (String[] args) 
    {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter number of test cases: ");
        int T=sc.nextInt();
        while(T-->0){
            System.out.println("Enter number: ");
            int n=sc.nextInt();
            int [] a=new int[100];
            int i=0;
            while(n!=0){
                a[i]=n%10;
                n=n/10;
                i++;
            }
            int sum=a[0]+a[i-1];
            System.out.println(sum);
        }
    }
}

File: FirstAndLastDigit.py
# If Give an integer N . Write a program to obtain the sum of the first and last digits of this number
#Solution Provided by CodingBroz 
T = int(input("Enter number of test cases: "))
while T > 0:
    n = (input("Enter number: "))
    reverse = n[::-1]
    print(reverse)
    reverse = int(reverse)
    n = int(n)
    first_digit = reverse % 10
    last_digit = n % 10
    sum = first_digit + last_digit
    print(sum)
    T = T - 1